examples/tests/backend_driver.py

```diff
@@ -501,21 +219,20 @@ def parse_options():
     else:
         python = [sys.executable]
```

Comment:
Since I can't comment directly on it, `sys.executable` should be the default on the `drive` function too.